sdk/tests/conftest.py

```diff
@@ -454,8 +453,8 @@ def generic_router_version(
         error="NONE",
         image="test.io/just-a-test/turing-router:0.0.0-build.0",
         routes=[generic_route for _ in range(2)],
-        default_route="http://models.internal/default",
-        default_route_id="control",
+        default_route=generic_route.endpoint,
```

Comment:
These tests were originally using a default route id that wasn't a part of the routes themselves. Now, the `_verify_default_route_exists` method added to `sdk/turing/router/config/router_config.py` will prevent that, so the route must be valid.

Reply:
Whoops, thanks a lot for adding this so more upfront checks are performed before sending any requests to the API!